assets/test/source/configs/source-invalid.hcl

```diff
@@ -0,0 +1,3 @@
+source {
```

Comment:
I don't love having separate configs in the repo if it can be avoided - you'll see in [my most recent PR](https://github.com/snowplow/snowbridge/pull/339/files) I actually started removing some of them & using the docs examples instead.

The ones for testing the config logic itself might be necessary - or at least not easy to remove. But it feels like for these ones we might be able to find a different approach. What do you think?

Reply:
Thanks @colmsnowplow ! Removed most of the test configs that were added. A couple that remain could be also removed with some refactor of `Init`, but sounded like out of scope for this PR.